data_pipeline/twitter_ads_api/twitter_ads_api_pipeline.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any
```

Comment:
should we have tests for this module?

Reply:
I did not find an avantage to add test for the current functions? I am planing to add a test in next PR as there will be different kind of functions to test which would make more sense then.

Reply:
I guess it would be that we passed around the values correctly.
Perhaps an end to end test? (I guess we would need to add the credentials to vault)

Reply:
yes can do when Scott is back to create one.